README.md

```
@@ -275,6 +306,6 @@ This project is built upon several excellent open-source projects:
 - [DexPilot](https://arxiv.org/abs/1910.03135) for vision-based teleoperation insights
 - [VisionProTeleop](https://github.com/Improbable-AI/VisionProTeleop) for Apple Vision Pro streaming
 
-## License
+## Contact
 
-MIT
+For any questions, please contact support@wuji.tech.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)